components/sandbox/toolchain-host-operator/base/monitoring/sandbox-registration-service-proxy.yaml

```diff
@@ -33,23 +33,23 @@ roleRef:
   name: registration-service-metrics-reader
 subjects:
 - kind: ServiceAccount
-  name: metrics-reader
-  namespace: toolchain-host-operator
+  name: registration-service-metrics-reader
+  namespace: system
 ---
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: sandbox-registration-service-proxy
-  namespace: toolchain-host-operator
+  namespace: system
 spec:
   endpoints:
     - interval: 15s
       scheme: http
       path: /metrics
       port: proxy-metrics
       bearerTokenSecret:
```

Comment:
For future: `bearerTokenSecret` seems to be deprecated, we should be using `authorization` instead.